Sources/SentryCrash/Recording/Tools/SentryCrashJSONCodec.c

```diff
@@ -342,8 +342,10 @@ sentrycrashjson_addUIntegerElement(
 {
     int result = sentrycrashjson_beginElement(context, name);
     unlikely_if(result != SentryCrashJSON_OK) { return result; }
-    char buff[30];
-    sprintf(buff, "%" PRIu64, value);
+#define SENTRY_UINT_BUFFER_SIZE 30
+    char buff[SENTRY_UINT_BUFFER_SIZE];
+    snprintf(buff, SENTRY_UINT_BUFFER_SIZE, "%" PRIu64, value);
+#undef SENTRY_UINT_BUFFER_SIZE
```

Comment:
l: for `addIntegerElement` and `addFloatingPointElemenet` we have this:
```C
char buff[50];
snprintf(buff, sizeof(buff), "%lg", value);    
```
Which is different from the `#define` approach. Can we use the same approach in all places? 
And if you choose to use the `#define` I prefer to have one for the file, instead of one every function.

Reply:
I like the `sizeof` approach better!